examples/api/lib/material/input_chip/input_chip.0.dart

```diff
@@ -14,7 +14,8 @@ class ChipApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      theme: ThemeData(colorSchemeSeed: const Color(0xff6750a4), useMaterial3: true),
+      theme: ThemeData(
+          colorSchemeSeed: const Color(0xff6750a4), useMaterial3: true),
```

Comment:
Looks like most of the diff just reformat the code. Can we revert them for now because it's not this PR's purpose.

Also please check [Style guide for flutter repo](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) when we reformat code so that the code style can be consistent. For example we usually give [2-character indentation](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) to list arguments:)

Reply:
Yes of course . It's just added by mistake ,this is the vscode thing.